receiver.py

```diff
@@ -0,0 +1,52 @@
+from time import sleep
+from typing import Union
+
+import daiquiri
+from celery import Celery
+from celery.events.state import Task, Worker
+
+from exporters import Exporter
+from utils import is_event_type_task
+
+logger = daiquiri.getLogger(__name__)
+
+
+class CeleryEventReceiver:
+    def __init__(self, broker: str):
+        self.broker = broker
+        self.celery_app = Celery(
+            broker=self.broker,
+        )
+        self.exporters: list[Exporter] = []
+        self.state = self.celery_app.events.State()
+
+    def attach(self, exporter):
+        self.exporters.append(exporter)
+
+    def notify(self, event: Union[Task, Worker]):
```

Comment:
Optional:

The interface of this class is ambiguous. If i have an event object, should i call .notify or .notify_event
Perhaps this can be an internal method if not called from outside, say `self._notify_exporters(event)`

Reply:
yep make sense will make the change. 
Ideally the user of this lib, won't be calling notify or notify_event. This class should ideally just take a configuration for filters (filter like type of events, type of task ... ) and would be only calling the run func.